lib/modules/manager/hermit/types.ts

```diff
@@ -0,0 +1,29 @@
+import type { SpawnSyncReturns } from 'child_process';
+import type { PackageDependency } from '../types';
+
+export interface HermitListItem {
+  Name: string;
+  Version?: string;
+  Channel?: string;
+}
+
+export interface UpdateHermitError<T>
+  extends SpawnSyncReturns<T>,
+    UpdateHermitResult {}
```

Comment:
what is this used for?

Reply:
This is the error type throw in `artifacts.ts`

Reply:
I think this is now obsolete and can be removed